src/views/footerView.jsx

```diff
@@ -117,6 +121,26 @@ export const ConferenceFooter = ({ isChatOpen, toggleChat }) => {
       : addNoiseSuppressionPlugin();
   }
 
+  async function raiseHand() {
+    if (!isHandRaised) {
```

Comment:
if else is not required, `metadata: JSON.stringify({ raiseHand: !isHandRaised })` can be done to simplify code